hack/linter/lint-paths.txt

```diff
@@ -17,8 +17,9 @@ pkg/network/pod/annotations
 pkg/network/vmispec
 pkg/storage/pod/annotations
 pkg/virtctl/clientconfig
-pkg/virtctl/credentials
```

Comment:
Please don't remove existing lines in `lint-paths.txt`.

Reply:
i just changed the order, so it appear alphabetically.